copulas/univariate/gaussian_kde.py

```diff
@@ -109,7 +107,7 @@ def percent_point(self, U):
         """
         self.check_fit()
 
-        return scipy.optimize.brentq(self._brentq_cdf(U), -1000.0, 1000.0)
```

Comment:
What if you're data is outside of the range [-1000.0, 1000.0]? For example, timestamps?